src/main/java/blip/tasks/Task.java

```diff
@@ -0,0 +1,93 @@
+package blip.tasks;
+
+import java.time.LocalDateTime;
+import blip.exceptions.*;
+import blip.parser.*;
+
+/**
+ * Represents a generic task.
+ */
+public abstract class Task {
+    /**
+     * Description of the task.
+     */
+        protected String description;
```

Comment:
Perhaps indentation could be set to 4 to match the other attributes of Task class?